### DISCUSSION TITLE
Funnel apha program

### DISCUSSION THREAD

**Discussion by mtosin** [hassio-addons/app-tailscale#142] [Ideas]
Hi all,
I hope you can help me to enable and expose automatically out HA over the internet with Tailscale.

this future can be achieved by being part of Funnel alpha program for the moment.

I would like to test and have a parameter in the HA addon configuration to start the service as per this doc https://tailscale.com/kb/1223/tailscale-funnel/

$ tailscale serve / proxy 3000
$ tailscale serve funnel on

**Comment by lmagyar** [comment]
You can test it with my fork: https://github.com/lmagyar/homeassistant-addon-tailscale

**Reply by mtosin** [reply]
wow, all works fine and validated!

great job, was planning to implement myself if not available :)

**Reply by mtosin** [reply]
hi,
i see you discontinued this fork.
I hope is now merged in default branch.

how to enable there the funnel?

**Reply by lmagyar** [reply]
It's not merged into the official add-on, but is available in my fork at: https://github.com/lmagyar/homeassistant-addon-tailscale

**Comment by mtosin** [comment]
Yes I see. thanks a lot.
great addon :)

Il Lun 24 Lug 2023, 13:23 Laszlo Magyar ***@***.***> ha
scritto:
